core/arch/opnd_shared.c

```diff
@@ -1980,8 +1984,9 @@ reg_is_extended(reg_id_t reg)
             (reg >= REG_START_16 + 8 && reg <= REG_STOP_16) ||
             (reg >= REG_START_8 + 8 && reg <= REG_STOP_8) ||
             (reg >= REG_START_x64_8 && reg <= REG_STOP_x64_8) ||
-            (reg >= REG_START_XMM + 8 && reg <= REG_STOP_XMM) ||
-            (reg >= REG_START_YMM + 8 && reg <= REG_STOP_YMM) ||
+            (reg >= DR_REG_START_XMM + 8 && reg <= DR_REG_STOP_XMM) ||
```

Comment:
Are you sure about this, 8 through 31?  How are *mm16-31 selected?
(Ditto the next two lines.)

Reply:
I think this function is a little bit legacy, returning whether registers are "extended" in the sense of 64 bit over 32 bit? That's what the description says. So yes, 32 bit xmm0 - xmm7, 64 bit xmm8 - xmm31.